src/styles/block/page.scss

```diff
@@ -0,0 +1,12 @@
+.page {
+  font-family: $font-family;
+  font-weight: 400;
+  font-size: 10px;
+  font-style: normal;
+  line-height: 14px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to order properties within a selector in a specific order: positioning, box model, typography, visual, misc. In your case, 'line-height' (visual) should come after 'font-size' (typography).